tests/tests-hz/src/main/java/com/hazelcast/simulator/tests/icache/IntByteICacheTest.java

```diff
@@ -55,6 +59,8 @@
     public KeyLocality keyLocality = KeyLocality.SHARED;
     // timeout in milliseconds for Cache asynchronous operations
     public int timeoutMs = 50;
+    public int backoffMs = 50;
```

Comment:
I would appreciate a short comment of this field. Why would we want to sleep during the exception handling?